src/bitcoin-cli.cpp

```diff
@@ -1028,6 +1080,64 @@ static void ParseGetInfoResult(UniValue& result)
         result_string += "\n";
     }
 
+    const double mempool_bytes{result["mempool_bytes"].get_real()};
+
+    if (mempool_bytes >0){
+
+    	result_string += strprintf("%sMempool:%s\n", MAGENTA, RESET);
+
+    	std::string fee_rate;
+    	std::string size_bar;
+    	double fee_size = 0;
+
+    	double size1 = result["feerate_dist1"].get_real();
+    	double size2 = result["feerate_dist2"].get_real();
+    	double size3 = result["feerate_dist3"].get_real();
+    	double size4 = result["feerate_dist4"].get_real();
+
+    	//Removing zero from numbers used in std::min to avoid division by zero in GetFeerateBar()
+    	auto const ignore_zero = [](auto const& a, auto const& b) -> bool {
+        	if(0 == a || 0 == b)
+        	{
+            	return false;
+        	}
+        	return a < b;
+    	};
+
+    	double max_size = std::max({size1, size2, size3, size4});
+    	double inc_size = std::min({size1, size2, size3, size4}, ignore_zero);
+
+    	for (int i = 1; i < 5; ++i) {
+
+        	switch (i)
+    		{
+    		case 1: fee_rate = "1-9       |";
+    		        fee_size = size1;
+        		break;
+    		case 2: fee_rate = "10-99     |";
+    		        fee_size = size2;
+        		break;
+    		case 3: fee_rate = "100-199   |";
+    		        fee_size = size3;
+        		break;
+        	case 4: fee_rate = "Above 200 |";
```

Comment:
This line makes me think that value `200` is not included neither in case 3 nor in case 4.

Reply:
200 is included in case 4. I wrote `Above 200` in short for `200 and Above 200`

Reply:
Yeah. Maybe one can use `≥ 200` (https://www.fileformat.info/info/unicode/char/2265/index.htm) instead of "Above 200" to make the meaning absolutely clear.